### PR TITLE
chore: reduce length of short descriptions

### PR DESCRIPTION
**Pull request by benoitf**
Reduce length of the short descriptions which were not short.

if agreement on the title, I'll create PR to update package.json of each repository of the extensions

fixes https://github.com/containers/podman-desktop/issues/6771

Change-Id: I3eb78a2100a232e3702ff9602ed73dceb1eae6ce

### CODE REVIEW

**Review by jeffmaury — APPROVED** [review]
LGTM.

Would rephrase

Best of Red Hat extensions for Podman Desktop

 as

Recommended Red Hat extensions for Podman Desktop

File: api/extensions.json [hunk 1]
```diff
@@ -649,7 +649,7 @@
       },
       "extensionName": "redhat-pack",
       "displayName": "Red Hat Extension Pack",
-      "shortDescription": "Get extensions of Red Hat for Podman Desktop",
+      "shortDescription": "Best of Red Hat extensions for Podman Desktop",
```
Comment (benoitf) [inline api/extensions.json:652]:
```suggestion
      "shortDescription": "Recommended Red Hat extensions for Podman Desktop",
```

**Review by jeffmaury — APPROVED** [review]

File: api/extensions.json [hunk 2]
```diff
@@ -311,7 +311,7 @@
       },
       "extensionName": "minikube",
       "displayName": "minikube",
-      "shortDescription": "Integration for minikube: a tool that makes it easy to run Kubernetes locally",
+      "shortDescription": "Run Kubernetes locally",
```
Comment (deboer-tim) [inline api/extensions.json:314]: A little too generic? This could apply to OpenShift Local or Kind too. I know it's obvious beside the display name but feels like it could use something like 'Run Kubernetes locally with minikube' or something that's unique about minikube.

**Review by deboer-tim — APPROVED** [review]

File: api/extensions.json (hunk 2 above)
Comment (benoitf) [reply api/extensions.json:314]: At first I used with minikube but then in the catalog you see the name 'Minikube' and the description is to run Kubernetes so I would say it's the combination of the two. It feels redundant to say it's with minikube as it's the name of the extension

The GitHub page of minikube says `Run Kubernetes locally`

File: api/extensions.json [hunk 3]
```diff
@@ -649,7 +649,7 @@
       },
       "extensionName": "redhat-pack",
       "displayName": "Red Hat Extension Pack",
-      "shortDescription": "Get extensions of Red Hat for Podman Desktop",
+      "shortDescription": "Recommended Red Hat extensions for Podman Desktop",
```
[the remark this answers is not in the document]
Comment (benoitf) [reply api/extensions.json:652]: at first it was not there but then people reported that it was not obvious it was for Podman Desktop and not for VS Code as we do extensions in `redhat-developer` project for VS Code

I'm fine removing it again

File: api/extensions.json [hunk 4]
```diff
@@ -177,7 +177,7 @@
       },
       "extensionName": "openshift-local",
       "displayName": "Red Hat OpenShift Local",
-      "shortDescription": "This extension provides a Red Hat OpenShift Local cluster and allows Podman Desktop to interact with it",
+      "shortDescription": "Run Red Hat OpenShift locally",
```
[the remark this answers is not in the document]
Comment (benoitf) [reply api/extensions.json:180]: I would replace OpenShift by MicroShift :-)

`Run MicroShift locally`
